examples/check-if-image-loaded/README.md

```diff
@@ -0,0 +1,20 @@
+# Check If an Image Has Loaded Without Errors
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to check whether an image on the page has loaded without errors.
+
+The tested page requests an image from the server. The server responds with the image after a timeout specified in the request options.
```

Comment:
The server responds with the image after a specified timeout.

Reply:
edited

Reply:
edited